src/wpscrack.c

```diff
@@ -162,7 +162,8 @@ static int reaver_usage(char *prog_name)
         fprintf(stderr, "\t-h, --help                      Show help\n");
         
         fprintf(stderr, "\nAdvanced Options:\n");
-	fprintf(stderr, "\t-p, --pin=<wps pin>             Use the specified pin (may be arbitrary string or 4/8 digit WPS pin)\n");
+	fprintf(stderr, "\t-p, --pin=<wps pin>             Use the specified pin (may be arbitrary string or 4/7/8 digit WPS pin)\n");
```

Comment:
4/7/8 change seems unrelated to this PR too

Reply:
IIRC that just computes the checksum automatically. in any case, this change should not added as an easter egg to this commit but should be a separate change.